### ISSUE TITLE
examples => webgl_points_nodes.html BUG

### ISSUE THREAD

**Issue by shihaoning** [mrdoob/three.js#24078]
**Describe the bug**

I want to create two sets of particles in the scene, with two maps with different frames (one 6x6, the other 8x8).But the spriteSheetCount setting for the second material added to the scene is invalid.

It seems that only the first spriteSheetCount is recognized in the scene.

**To Reproduce**

1. Create two Points using different PointsNodeMaterial（different textures for different frames）
2. See error

***Code***

```js
 const spriteSheetCount1 = new Nodes.ConstNode(new THREE.Vector2(6, 6));

 /**Not work */
 const spriteSheetCount2 = new Nodes.ConstNode(new THREE.Vector2(8, 8));
```

**Expected behavior**

Scenarios can support multiple sets of particles, using several different frames of map

**Screenshots**

![firetorch_1](https://user-images.githubusercontent.com/28341337/168951859-dcbb192e-89f9-4ed7-a831-f446276dffee.jpg)
![fire3](https://user-images.githubusercontent.com/28341337/168951914-8f9a8581-8261-40a9-8d5c-eadba057bbad.jpg)
![image](https://user-images.githubusercontent.com/28341337/168951951-e2dc06ab-e7d6-409a-ac72-909c78155a17.png)

**Platform:**

 - Device: [Desktop, Mobile]
 - OS: [Windows, MacOS, Linux, Android, iOS]
 - Browser: [Chrome, Firefox, Safari, Edge]
 - Three.js version: [dev, r140]

**Comment by Mugen87** [comment]
Can you please demonstrate the issue with a [live example](https://jsfiddle.net/7u84j6kp/)?

**Comment by shihaoning** [comment]
Sorry, I can't access jsfiddle.net here
But I can provide an offline demo

[webgl_points_nodes.zip](https://github.com/mrdoob/three.js/files/8715135/webgl_points_nodes.zip)

> Can you please demonstrate the issue with a [live example](https://jsfiddle.net/7u84j6kp/)?

**Comment by sunag** [comment]
> Can you take a look at this, or am I using it incorrectly? thanks

@shihaoning thanks for reporting and sorry the late, fixed here https://github.com/mrdoob/three.js/pull/24096

**Comment by shihaoning** [comment]
> @shihaoning thanks for reporting and sorry the late, fixed here #24096

cool!
![image](https://user-images.githubusercontent.com/28341337/169630547-aec89708-6626-4d60-9ef4-83ad6e802714.png)
